recipes/r-geiger/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   sha256: 9153047b608d652821251206d1450bb3f517c8884379f498a695315574ae001d
 
 build:
-  merge_build_host: True  # [win]
```

Comment:
@dbast This was not `noarch` and failed, because its `r-subplex` dependency is not build for win. In this case, the `skip: True  # [win]` can stay?

Reply:
Correct. This (r-geiger) is a compiled package and has to be built for the architecture where it should run... if a dependency as in this case `r-subplex` is not available, the architecture has to be skipped.